.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-*"
+
+  pull_request:
```

Comment:
I don't know if we want to release on a PR, unless I am missing something which very well might be the case

Reply:
We were discussing to have a build for every pr, so that if anyone wants to test the extension can jut download and install.

Reply:
just*

Reply:
May be I need two workflows. One for PRs and one one for releases. Thanks for the question

Reply:
also this only uploads the vsix package as artifacts to the ci runs. eg: https://github.com/savitharaghunathan/editor-extensions/actions/runs/11111525940. @shawn-hurley wdyt? we can chat more if you think it shouldnt be this way. May be I am missing something

Reply:
I think that all makes a ton of sense! I very much think then that is a good idea and something that I would for sure use